src/batches/listBatches.tsx

```diff
@@ -204,24 +214,24 @@ function ListBatches({
 
   return (
     <div>
+      <div className="create-batches-overlay ">
+        <div
+          className="create-batch-overlay"
```

Comment:
name these better? not a fan of having a
"create-batches-overlay" and "create-batch-overlay"
and they do different things.

Reply:
resolved